common/src/lib.rs

```diff
@@ -266,6 +275,20 @@ pub fn return_correct_icon(file_name: &str) -> Icon {
     Icon::DocumentQuestion
 }
 
+pub fn get_file_type(file_name: &str) -> FileType {
+    if is_video(file_name) {
+        FileType::Video
+    } else if is_audio(file_name) {
+        FileType::Audio
+    } else if is_doc(file_name) {
+        FileType::Doc
+    } else if is_lang_file(file_name) {
+        FileType::Code
+    } else {
+        FileType::Image
```

Comment:
What if a file is not an image? Would it be better to have a generic type that would be the default?